Gemfile

```diff
@@ -63,6 +63,7 @@ gem 'will_paginate-bootstrap'
 gem 'yt'
 gem 'yui-compressor'
 gem 'youtube_rails'
+gem 'google-api-client', '0.8'
```

Comment:
I do, however agree that it is a problem that google has not updated their samples. 
https://github.com/youtube/api-samples/issues/63#issuecomment-318918531